2024/shorts/mandelbrot/german/sentence_translations.json

```
@@ -32,15 +32,15 @@
   ]
  },
  {
-  "translatedText": "Abhängig von der Wahl dieses Werts c bleibt die Folge manchmal begrenzt und manchmal geht sie in die Unendlichkeit über.",
+  "translatedText": "Abhängig von der Wahl dieses Werts c bleibt die Folge manchmal begrenzt und manchmal bricht sie aus ins Unendliche.",
   "input": "Depending on the choice for that value c, sometimes the sequence stays bounded, and sometimes it blows up to infinity.",
   "time_range": [
    35.56,
    41.54
   ]
  },
  {
-  "translatedText": "Wenn Sie alle Werte von c, die dazu führen, dass dieser Prozess begrenzt bleibt, schwarz einfärben und auf die anderen Werte einen Farbverlauf anwenden, wobei die Farbe davon abhängt, wie schnell der Prozess bis ins Unendliche ansteigt, erhalten Sie diese ikonische Niere -mit-Blasen-Form.",
+  "translatedText": "Wenn man alle Werte von c, die dazu führen, dass dieser Prozess begrenzt bleibt, schwarz lässt, und die anderen Werten als Farbverlauf darstellt, wobei die Farbe davon abhängt, wie schnell die Iteration gegen unendlich geht, ergibt sich diese ikonische Herzkurve-mit-Blasen-Form.",
```

I think, the words "kurve" and "Form" are a little bit redundant. In my humble opinion "Herz-mit-Blasen-Form" would be better, even though Wikipedia says: "cardioid" means "Herzkurve" (https://de.wikipedia.org/wiki/Kardioide).

---

Remove the "n" from "Werten":
```suggestion
  "translatedText": "Wenn man alle Werte von c, die dazu führen, dass dieser Prozess begrenzt bleibt, schwarz lässt, und die anderen Werte als Farbverlauf darstellt, wobei die Farbe davon abhängt, wie schnell die Iteration gegen unendlich geht, ergibt sich diese ikonische Herzkurve-mit-Blasen-Form.",
```